### PR TITLE
Create EIP-45000.json

### PR DESCRIPTION
**Pull request by defido**
Add Autobahn network

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
You need to rebase to master so your icon is found by the CI

**Review by ligi — CHANGES_REQUESTED** [review]
the prefix must be eip155 - not just EIP

**Comment by defido** [comment]
ok completed

File: _data/chains/eip155-45000.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Autobahn Network",
+  "chain": "BNB",
+  "network": "mainnet",
+  "rpc": [
+    "https://rpc.autobahn.network",
```
Comment (ligi) [inline _data/chains/eip155-45000.json:6]:
```suggestion
    "https://rpc.autobahn.network"
```

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `bnb` more than once

**Comment by defido** [comment]
changed

**Review by ligi — APPROVED** [review]

**Comment by defido** [comment]
Legend, thanks!

**Pull request merged**